locales/th/product-registration.json

```diff
@@ -0,0 +1,32 @@
+{
+  "register": "ลงทะเบียนสินค้าของคุณ",
+  "productImage": "รูปภาพสินค้า",
+  "addThreeImages": "เพิ่มได้ทั้งหมด 3 รูป",
+  "name": "ชื่อสินค้า",
+  "price": "ราคาที่เหมาะสม (บาท)",
+  "color": "สี",
+  "size": "ขนาด",
+  "category": "ประเภทสินค้า",
+  "deliveryOption": "ตัวเลือกการจัดส่ง",
+  "footwear": "รองเท้า",
+  "shirt": "เสื้อ",
+  "accessory": "เครื่องประดับ",
+  "postal": "ไปรษณีย์",
+  "meetUp": "นัดรับ",
+  "both": "ไปรษณีย์และนัดรับ",
+  "description": "รายละเอียดสินค้า",
+  "submit": "ลงขายสินค้า",
+  "namePlaceholder": "เช่น Nike Air Max, Adidas Ultraboost",
+  "pricePlaceholder": "เช่น 290, 3500.50",
```

Comment:
Same as this [comment](https://github.com/Cobda/cobda-web/pull/55/files#r619825106)